scripts/verify.ts

```diff
@@ -0,0 +1,30 @@
+import { run, ethers, upgrades, network, baseURI, proxy } from "hardhat";
+
+async function main() {
+  console.log(`\nVerifying on '${network.name}'...`);
+
+  // Get base URI
+  if (baseURI === undefined || baseURI === "") {
+    throw Error(`missing baseURIs entry for '${network.name}'`);
+  }
+
+  // Ensure deployments
+  if (proxy === "") {
+    throw Error(`no proxy entry for '${network.name}'`);
+  }
+
+  // Verify implementation
+  const tables = (await ethers.getContractFactory("TablelandTables")).attach(
+    proxy
+  );
+  const impl = await upgrades.erc1967.getImplementationAddress(tables.address);
+  await run("verify:verify", {
```

Comment:
We can use this to verify the implementation contracts once this code base matches what is deployed.